frontend/www/js/omegaup/polyfills.js

```diff
@@ -59,3 +59,19 @@ if (!String.prototype.padStart) {
     }
   };
 }
+
+// From https://medium.com/trabe/using-promise-allsettled-now-e1767d43e480
+Promise.allSettled = promises =>
```

Comment:
nit:

```suggestion
if (window.Promise && !Promise.allSettled) {
  Promise.allSettled = promises =>
```

la idea de los polyfills es que únicamente deben declararse si no está soportado nativamente.

Reply:
Listo